libraries/stdlib/src/kotlin/text/StringsJVM.kt

```diff
@@ -137,8 +137,8 @@ public inline fun String.Companion.format(format: String, vararg args: Any?): St
 public inline fun String.format(locale: Locale, vararg args : Any?) : String = java.lang.String.format(locale, this, *args)
 
 /**
- * Uses this string as a format string and returns a string obtained by substituting the specified arguments,
- * using the default locale.
+ * Uses this string as a format string and returns a string obtained by substituting the specified arguments, using
```

Comment:
I've spotted another copy-paste related issue: it uses not this string, but the provided [format] string.
Could you improve these two overloads' docs as well?

Reply:
Sure thing! Should I squash commits?

Reply:
Done